test/schemas/form/Person.ChangeDOB.schema

```diff
@@ -0,0 +1,27 @@
+Form({
+  Args: {
+    OldDOB: { field: 'DOB' },
+    NewDOB: { domain: 'DateTime' },
+    LastName: { field: 'FullName.LastName' },
+  },
+  Markup: {
+    Tabs: {
+      control: 'tabs',
+      children: {
+        tab: {
+          control: 'tab',
+          children: {
+            group: {
+              control: 'group',
```

Comment:
We need real control names from UI group @alinkedd

Reply:
It seems to me that this already goes far beyond the `metaschema`. Forms are a part of the `MIA framework`

Reply:
The same can be said about #171

Reply:
IIUC it is not, metaschema only provides a way to load schemas, processing of a schema may or may not be proprietary, but if we intend to load something by metaschema, we need to add a way to load it and test that it is loaded correctly.

Reply:
@tshemsedinov 
> We need real control names from UI group

Form Layout is not a subject of this PR, any data contained here is just an example of a structure we agreed upon so that `Layout` or `Markup` props are not an empty object.

Reply:
Ok, it's just a stub